src/test/fuzz/miniscript.cpp

```diff
@@ -777,7 +780,11 @@ NodeRef GenNode(F ConsumeNode, Type root_type, bool strict_valid = false) {
             // Fragment/children have not been decided yet. Decide them.
             auto node_info = ConsumeNode(type_needed);
             if (!node_info) return {};
-            // Update predicted resource limits.
+            // Update predicted resource limits. Since every leaf Miniscript node is at least one
+            // byte long, we move one byte from each child to their parent. A similar technique is
+            // used in the miniscript FromScript parser.
```

Comment:
Sorry to block just for a code comment.. But it's in `Parse()` (called by `FromString()`) that we have this logic, not `FromScript` (where we can just check the script size isn't larger than `3600`).